js/util/misc.ts

```diff
@@ -64,3 +64,5 @@ export function urlParam(name: string): string | null{
     return null;
   }
 }
+
+export type SvgIcon = React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
```

Comment:
How would you feel about putting this into its own file so that clients would use
```
import { SvgIcon } from "./svg-icon";
```
instead of
```
import { SvgIcon } from "./misc";
```
? Seems more portable to other projects, for instance.

Reply:
I like that idea.  I'll end up needing it in activity player too.  Fixed.